src/hooks/withQuery.tsx

```diff
@@ -0,0 +1,20 @@
+import { useQuery, UseQueryOptions } from 'react-query'
+import ky from 'ky'
+
+function withQuery<T>(
+  queryKey: string,
```

Comment:
ReactQuery exposes a QueryKey type
```suggestion
  queryKey: QueryKey,
```